v4/feature/profile/src/main/java/exchange/dydx/trading/feature/profile/rewards/DydxRewardsFaqsView.kt

```diff
@@ -42,7 +42,6 @@ object DydxRewardsFaqsView : DydxComponent {
                     localizer = MockLocalizer(),
                     title = "FAQs",
                     learnMoreText = "Learn more",
-                    link = "https://dydx.exchange",
```

Comment:
should this be replaced with the actual link? Would this just leave a `Learn More` that does not navigate?

Reply:
ah is this a preview file?

Reply:
Yeah